utils/_context/_scenarios/integration_frameworks.py

```diff
@@ -44,27 +51,40 @@ def configure(self, config: pytest.Config):
 
         framework, framework_version = weblog.split("@", 1)
 
+        # Handle weblog language name suffix needed for weblog definitions
+        # e.g., "openai-py" -> "openai", "openai-js" -> "openai"
+        framework_dir = framework.rsplit("-", 1)[0] if "-" in framework else framework
+
+        self._set_dd_trace_integrations_enabled(library)
+
         self._test_client_factory = FrameworkTestClientFactory(
             library=library,
             framework=framework,
             framework_version=framework_version,
             container_env=self.environment,
-            container_volumes={f"./utils/build/docker/{library}/{framework}_app": "/app/integration_frameworks"},
+            container_volumes={f"./utils/build/docker/{library}/{framework_dir}_app": "/app/integration_frameworks"},
         )
 
         self._test_agent_factory.configure(self.host_log_folder)
         self._test_client_factory.configure(self.host_log_folder)
 
-        # Set library version - for now use a placeholder, will be updated after building
-        self._library = ComponentVersion(library, "0.0.0")
-        logger.debug(f"Library: {library}, Framework: {framework}=={framework_version}")
-
         if self.is_main_worker:
             # Build the framework test server image
             self._test_client_factory.build(github_token_file=config.option.github_token_file)
             self._test_agent_factory.pull()
             self._clean()
 
+        # Extract library version from built container
+        output = get_docker_client().containers.run(
+            self._test_client_factory.tag,
+            remove=True,
+            command=["./system_tests_library_version.sh"],
+            volumes=compute_volumes(self._test_client_factory.container_volumes),
+        )
+
+        self._library = ComponentVersion(library, output.decode("utf-8"))
+        logger.debug(f"Library: {library}, Framework: {framework}=={framework_version}, Version: {self._library}")
```

Comment:
A very useful data to have in stdout the library version. Here is how to achieve that : 


```suggestion
        logger.debug(f"Library: {library}, Framework: {framework}=={framework_version}, Version: {self._library}")
        
    def get_warmups(self):
        result = super().get_warmups()
        result.append(lambda: logger.stdout(f"Library: {self.library}"))

        return result
```